src/Audit/StackMetrics.php

```diff
@@ -20,10 +20,40 @@
  *  default = {"web-cpu", "web-memory"}
  * )
  * @Param(
+ *  name = "chart-type",
+ *  description = "The type of graph, either bar or line.",
+ *  type = "string",
+ *  default = "bar"
+ * )
+ * @Param(
+ *  name = "chart-height",
+ *  description = "The height of the graph in pixels.",
+ *  type = "integer",
+ *  default = "250"
+ * )
+ * @Param(
+ *  name = "chart-width",
+ *  description = "The width of the graph in pixels.",
+ *  type = "integer",
+ *  default = "400"
+ * )
+ * @Param(
  *  name = "y-axis-label",
  *  description = "Custom label for the y-axis.",
  *  type = "string",
- *  default = {"Percentage"}
```

Comment:
I'm forgetting why they curly braces were here and what they mean.

Reply:
If a policy doesn't provide this parameter, the default is used and the curly braces turns it into an array rather than a string. I was getting PHP warnings until I removed those.